7_react/react1/week1/ToDoList/src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import "./index.css";
+import NewTodoList from "./Components/NewTodoList.jsx";
+import TaskInput from "./Components/TaskInput.jsx";
+
+function App() {
+  const newTodo = [
```

Comment:
static variables should be inside of React components, please move it above `function App() {`